<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_EditTaskUser</class>
 <widget class="QDialog" name="D_EditTaskUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование задачи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Наименование АВП:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelNameAVP">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Мажор</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>ФИО, оператора:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxUser">
     <item>
      <property name="text">
       <string>Петров С.С.</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Приоритет:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxPriority">
     <item>
      <property name="text">
       <string>Низкий</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Текущий статус:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelCurrentStatus">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Назначить статус:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBoxStatus">
     <item>
      <property name="text">
       <string>В работе</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Процент готовности:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBoxPercent">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Дата выполнения:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxAttribute">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="6">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Взят в аренду</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Куплен</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>дней</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Платный</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Бесплатный</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Нарушения</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidgetViolation">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>0</number>
        </attribute>
        <column>
         <property name="text">
          <string>Наименование</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Описание нарушения</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Изображение</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>217</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Экспертиза:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonPreviewExpertise">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/attach2.png</normaloff>:/icons/icons/attach2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>170</height>
      </size>
     </property>
     <property name="title">
      <string>Комментарии</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEditComment">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Нет нарушений&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>613</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApply">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Применить</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/ok.png</normaloff>:/icons/icons/ok.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/cancel.png</normaloff>:/icons/icons/cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonExit">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/info.png</normaloff>:/icons/icons/info.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxUser</tabstop>
  <tabstop>comboBoxStatus</tabstop>
  <tabstop>spinBoxPercent</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonExit</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonApply</sender>
   <signal>clicked()</signal>
   <receiver>D_EditTaskUser</receiver>
   <slot>slotApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>D_EditTaskUser</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>D_EditTaskUser</receiver>
   <slot>slotExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonPreviewExpertise</sender>
   <signal>clicked()</signal>
   <receiver>D_EditTaskUser</receiver>
   <slot>slotPreviewExpertise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>D_EditTaskUser</receiver>
   <slot>slotValueChangedDay(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>D_EditTaskUser</receiver>
   <slot>slotInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotApply()</slot>
  <slot>slotCancel()</slot>
  <slot>slotExit()</slot>
  <slot>slotPreviewExpertise()</slot>
  <slot>slotValueChangedDay(int)</slot>
  <slot>slotInfo()</slot>
 </slots>
</ui>
